demisto_sdk/commands/common/hook_validations/integration.py

```diff
@@ -496,3 +499,41 @@ def is_valid_feed(self):
                 print_error(Errors.feed_wrong_from_version(self.file_path, from_version))
                 return False
         return True
+
+    def is_fetch_params_exist(self) -> bool:
+        """
+        validate that all required fields in integration that have fetch incidents are in the yml file.
+        Returns:
+            bool. True if the integration is defined as well False otherwise.
+        """
+        return_value = True
+        if self.current_file.get('script', {}).get('isfetch') is True:
+            params = [_key for _key in self.current_file.get('configuration', [])]
+            for param in FETCH_REQUIRED_PARAMS:
+                if param not in params:
+                    print_error(f'Integration with fetch-incidents was detected '
+                                f'("isfetch:  true" was found in the YAML file).'
+                                f'\nA required parameter is missing or malformed in the file {self.file_path}, '
+                                f'the param is:\n{param}')
+                    return_value = False
+
+        return return_value
+
+    def is_feed_params_exist(self) -> bool:
+        """
+        validate that all required fields in integration that have fetch incidents are in the yml file.
+        Returns:
+            bool. True if the integration is defined as well False otherwise.
+        """
+        return_value = True
+        if self.current_file.get('script', {}).get('feed') is True:
```

Comment:
if you all it from is valid feed, the whole function should be called after it was checked that this is a feed, so you will not need this check here.